src/router/sidebarActions.js

```diff
@@ -1,61 +1,93 @@
-export const sidebarActionOptions = [
+const sidebarActionOptions = [
   {
     title: 'Back to Dashboard',
     icon: 'pi pi-arrow-left',
     buttonLink: { name: 'Home' },
     requiresSuperAdmin: false,
+    requiresAdmin: false,
+    project: 'ALL',
   },
   {
     title: 'List organizations',
     icon: 'pi pi-list',
     buttonLink: { name: 'ListOrgs' },
     requiresSuperAdmin: false,
+    requiresAdmin: false,
+    project: 'ALL',
   },
   {
     title: 'Create organization',
     icon: 'pi pi-database',
     buttonLink: { name: 'CreateOrgs' },
     requiresSuperAdmin: true,
+    requiresAdmin: true,
+    project: 'ALL',
   },
   {
     title: 'Register students',
     icon: 'pi pi-users',
     buttonLink: { name: 'RegisterStudents' },
     requiresSuperAdmin: true,
+    requiresAdmin: true,
+    project: 'ROAR',
   },
   {
     title: 'Register administrator',
     icon: 'pi pi-user-plus',
     buttonLink: { name: 'CreateAdministrator' },
     requiresSuperAdmin: true,
+    project: 'ALL',
   },
   {
     title: 'Create administration',
     icon: 'pi pi-question-circle',
     buttonLink: { name: 'CreateAdministration' },
     requiresSuperAdmin: true,
+    requiresAdmin: true,
+    project: 'ALL',
   },
   {
     title: 'Register Task',
     icon: 'pi pi-pencil',
     buttonLink: { name: 'RegisterGame' },
     requiresSuperAdmin: true,
+    requiresAdmin: true,
+    project: 'ALL',
+  },
+  {
+    title: 'Register Users',
+    icon: 'pi pi-user-plus',
+    buttonLink: { name: 'Register Users' },
+    requiresSuperAdmin: true,
+    requiresAdmin: true,
+    project: 'LEVANTE',
   },
 ];
 
-export const getSidebarActions = (isSuperAdmin = false, includeHomeLink = true) => {
-  const actions = sidebarActionOptions.filter((action) => {
-    if (action.requiresSuperAdmin && !isSuperAdmin) {
-      return false;
-    }
-    return true;
-  });
-  const actionsWithHomeLink = actions.filter((action) => {
-    if (!includeHomeLink && action.buttonLink.name === 'Home') {
-      return false;
-    }
-    return true;
-  });
+export const getSidebarActions = (isSuperAdmin = false, isAdmin = false, includeHomeLink = true) => {
```

Comment:
We discussed reformatting the arguments of this function to be a single object with keys `isSuperAdmin`, `isAdmin`, and `includeHomeLink`

Reply:
We also discussed implementing the filter for project in the sidebar actions, I think you already started that though